samples/angular/sitecore/config/JssAngularWeb.config

```diff
@@ -7,28 +7,27 @@
   Normally `jss deploy config` can do this for local development. To manually deploy, or to deploy via CI,
   this file can be placed in the `App_Config/Include` folder, or a subfolder of it, within the Sitecore site.
 -->
-<configuration xmlns:patch="http://www.sitecore.net/xmlconfig/" xmlns:set="http://www.sitecore.net/xmlconfig/set/" xmlns:role="http://www.sitecore.net/xmlconfig/role/">
+<configuration xmlns:patch="http://www.sitecore.net/xmlconfig/"
+               xmlns:set="http://www.sitecore.net/xmlconfig/set/"
+               xmlns:role="http://www.sitecore.net/xmlconfig/role/">
   <sitecore>
     <settings>
-      <!--
-        PROXY CONFIGURATION
+      <!-- PROXY CONFIGURATION
         When running JSS with the headless proxy (node-headless-ssr-proxy),
         the original IP address of clients is obscured by the proxy.
         This setting tells Sitecore to read the forwarded header the proxy sends,
-        thus making analytics track the correct original client IP address.
-      -->
-      <setting name="Analytics.ForwardedRequestHttpHeader" set:value="X-Forwarded-For" />
+        thus making analytics track the correct original client IP address. -->
+      <setting name="Analytics.ForwardedRequestHttpHeader"
```

Comment:
Looks like Prettier is still mashing the config formatting a bit. This `<setting />` shouldn't be multi-line, and then for all of the block comments, I'd stick with how they were (kind of a Sitecore standard) with
```
<!--
  Heading
  Description ...
  ...
-->
```
vs
```
<!-- Heading
  Description ...
  ... -->
```

Reply:
Noted. XML gods have been appeased.